src/scenes/Editor/index.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import AceEditor from 'react-ace';
+import emailjs from 'emailjs-com';
+import './style.scss';
+
+import 'ace-builds/src-noconflict/mode-html';
+import 'ace-builds/src-noconflict/theme-monokai';
+
+export default class Editor extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const starterCode = `<html>
+  <head>
+    <title>Code in the Dark</title>
+
+    <style>
+
+    </style>
+  </head>
+
+  <body>
+
+  </body>
+</html>
+`;
+
+    this.state = {
+      name: '',
+      problem: '',
+      codeValue: starterCode,
+      changeSinceSubmit: true,
+      successfulSubmit: true,
+      statusMessage: '',
+    };
+  }
+
+  onChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
+  }
+
+  onEditorChange = newCode => {
+    this.setState({
+      changeSinceSubmit: true,
+      codeValue: newCode,
+    });
+  }
+
+  submit = () => {
+    if (!this.state.changeSinceSubmit && this.state.successfulSubmit) {
+      this.setState({
+        statusMessage: 'No changes since last submit',
+      });
+
+      return;
+    }
+
+    const params = {
+      name: this.state.name,
+      problem: this.state.problem,
+      message_body: this.state.codeValue,
+    };
+
+    const templateId = process.env.REACT_APP_EMAIL_TEMPLATE;
+    const userId = process.env.REACT_APP_EMAIL_USER;
+
+    emailjs.send('gmail', templateId, params, userId)
+      .then(() => {
+        this.setState({
+          changeSinceSubmit: false,
+          statusMessage: 'Successful submission',
+        });
+      }, error => {
+        this.setState({
+          changeSinceSubmit: false,
+          statusMessage: error,
+        });
+      });
+  }
+
+  render() {
+    const {
+      statusMessage,
+      changeSinceSubmit,
+      successfulSubmit,
+      codeValue,
+    } = this.state;
+
+    const displayStatus = !successfulSubmit || !changeSinceSubmit;
+
+    return (
+      <div>
+        <h1>Code in the Dark!</h1>
+
+        <div className="content">
```

Comment:
Give this some left margin, judging by the picture it's touching the edge of the screen